api/app/Models/AwardExperience.php

```diff
@@ -51,4 +51,10 @@ public function skills(): MorphToMany
             ->withPivot('details')
             ->as('experience_skill_pivot');
     }
+
+    public function poolCandidates(): MorphToMany
```

Comment:
Unless there's a reason this relationship would be particularly useful, let's remove it from all the Experience models. I don't think we'll need to go from experience to candidate in this direction, and I think its a bit confusing how/why it actually connects.

Reply:
I still think you should remove these from the Experience models. I think these relationships are confusing, in this direction.

Reply:
Okay sure, if you would like it so. I thought it might be helpful for model files to map all relations between one another.

Reply:
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/6379/commits/51800a25447eae0df10392bf26404b9df91a4bbe